### ISSUE TITLE
option can't be used as a field name in maven-plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [hanson-opus/protostuff#131]
```
Following Protobuff would generate MessageWithOption class with errors

package com.protostuff.error;
option optimize_for = LITE_RUNTIME;

option java_package = "com.protostuff.error";
option java_outer_classname = "MessageWithOption";

message MessageWithOption {
    optional string option = 1;
}
```

Original issue reported on code.google.com by `mavar...@gmail.com` on 9 Aug 2012 at 9:06

**Comment by GoogleCodeExporter** [comment]
```
fixed @ rev 1551.  Thanks
```

Original comment by `david.yu...@gmail.com` on 29 Aug 2012 at 8:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Yea this had been fixed for 1.0.8.
See http://code.google.com/p/protostuff/issues/detail?id=131

In the mean time, you can build the compiler jar yourself via:
cd protostuff-parser && mvn install
cd protostuff-compiler && mvn -Pjwd package

The output jar can be found in 
protostuff-compiler/target/protostuff-compiler-$version-jarjar.jar
```

Original comment by `david.yu...@gmail.com` on 16 Mar 2013 at 2:48

**Comment by GoogleCodeExporter** [comment]
```
Issue 142 has been merged into this issue.
```

Original comment by `david.yu...@gmail.com` on 29 May 2013 at 11:15
